### PR TITLE
[IMP] stock, mrp: popover widget and extenstions to owl

### PR DESCRIPTION
**Pull request by AhmedAmirK**
This commit converts `popover_widget`, `stock_rescheduling_popover`
and `mrp_workorder_popover` to OWL

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/models/mrp_production.py [hunk 1]
```diff
@@ -400,17 +401,20 @@ def _compute_delay_alert_date(self):
...
-                'delay_alert_date': format_datetime(self.env, production.delay_alert_date, dt_format=False),
-                'late_elements': [{
+            late_elms = [{
                         'id': late_document.id,
                         'name': late_document.display_name,
                         'model': late_document._name,
                     } for late_document in production.move_raw_ids.filtered(lambda m: m.delay_alert_date).move_orig_ids._delay_alert_get_documents()
                 ]
+            production.json_popover = json.dumps({
+                'popoverTemplate': 'stock.PopoverStockRescheduling',
```
Comment (amoyaux) [inline addons/mrp/models/mrp_production.py:413]: You still need the template in the json?

File: addons/mrp/models/mrp_production.py [hunk 2]
```diff
@@ -400,17 +401,20 @@ def _compute_delay_alert_date(self):
...
-                'popoverTemplate': 'stock.PopoverStockRescheduling',
-                'delay_alert_date': format_datetime(self.env, production.delay_alert_date, dt_format=False),
-                'late_elements': [{
+            late_elms = [{
                         'id': late_document.id,
                         'name': late_document.display_name,
                         'model': late_document._name,
                     } for late_document in production.move_raw_ids.filtered(lambda m: m.delay_alert_date).move_orig_ids._delay_alert_get_documents()
                 ]
+            production.json_popover = json.dumps({
```
Comment (amoyaux) [inline addons/mrp/models/mrp_production.py:412]: You still display the json_popover field somewhere? Since you are now using planning_issues in the view

Comment (AhmedAmirK) [reply addons/mrp/models/mrp_production.py:412]: planning_issues is just used in the listview, json_popover is still used in the formview. This is to hide the JSON value being displayed when hovering the field in listview

**Comment by amoyaux** [comment]
robodoo r+

**Pull request closed without merging**